src/EditorFeatures/VisualBasicTest/Diagnostics/SimplifyTypeNames/SimplifyTypeNamesTests.vb

```diff
@@ -2421,6 +2421,19 @@ End Class",
 End Class")
         End Function
 
+        <WorkItem(19498, "https://github.com/dotnet/roslyn/issues/19498")>
+        <Fact(), Trait(Traits.Feature, Traits.Features.CodeActionsSimplifyTypeNames)>
+        Public Async Function TestMyClassShouldNotBeRemoved() As Task
+            Await TestMissingInRegularAndScriptAsync(
+"Class SomeType
+    Overridable Sub Test()
```

Comment:
:bulb: It would be good to add another case where `Test` is *not* marked as `Overridable`. In this case, `MyClass` can be removed. @CyrusNajmabadi what's the general approach for cases like this? Also would we normally want to account for `Sealed` here?

:bulb: In addition to `Overridable`, I believe there are other modifiers that have the same impact: `MustInherit`, `Overrides`. @DustinCampbell do you know more?